pysat/tests/instrument_test_class.py

```diff
@@ -117,6 +117,19 @@ def generate_instrument_list(package=None):
 class InstTestClass():
     """Provides standardized tests for pysat instrument libraries.
     """
+    def setup(self):
+        self.module_attrs = ['platform', 'name', 'tags', 'sat_ids',
+                             'load', 'list_files', 'download']
+        self.inst_attrs = ['tag', 'sat_id', 'acknowledgements', 'references']
+        self.inst_callable = ['load', 'list_files', 'download', 'clean',
+                              'default']
+        self.attr_types = {'platform': str, 'name': str, 'tags': dict,
+                           'sat_ids': dict, 'tag': str, 'sat_id': str,
+                           'acknowledgements': str, 'references': str}
+
+    def teardown(self):
+        del self.inst_attrs, self.inst_callable, self.inst_types
+        del self.module_attrs
```

Comment:
`setup` and `teardown` are set by the class inheriting from `InstTestClass`.  In this case, it's `TestInstruments` in test_instruments.py.  This will be different for each package that inherits these tests.  Running tests locally, declaring things here just disappear, and the test says that `self.module_attrs` does not exist.

I see two options: define these as variables (suggestion below), or moving to the individual functions (which is not as portable).
```suggestion
    module_attrs = ['platform', 'name', 'tags', 'sat_ids',
                    'load', 'list_files', 'download']
    inst_attrs = ['tag', 'sat_id', 'acknowledgements', 'references']
    inst_callable = ['load', 'list_files', 'download', 'clean', 'default']
    attr_types = {'platform': str, 'name': str, 'tags': dict,
                  'sat_ids': dict, 'tag': str, 'sat_id': str,
                  'acknowledgements': str, 'references': str}
```

Reply:
Shouldn't these be `self.module_attrs`, etc.?

Reply:
But yes, wasn't sure if this would be the better way to do things and am fine changing it thusly.

Reply:
OK.  If we define them here, they are attached to the class automatically (since we are defining under the class and not one of the methods).